### ISSUE TITLE
Release 1/May Performance/Stability Issue

### ISSUE THREAD

**Issue by rbdevr** [jitsi/jitsi-meet#6581]
Good evening,

I’ve updated my server with the latest version and I’ve noticed a huge performance drop. The server cant handle more than 2 users and some cases the user cant connect to the website.

I’m running on AWS EC2 Ubuntu 18.04 standard setup. It was running ok before this version.

I tried a few things but in the end, I’ve decided to rollback to the version 10/April.

Are you guys facing the same issue. Could you have a look at it.

Cheers

**Comment by biolim** [comment]
I have the same problem. Rolled back the old version

**Comment by cmdares** [comment]
I have the same problem

**Comment by saghul** [comment]
Do you have the package versions you were running before and after?

**Comment by rbdevr** [comment]
jitsi-meet packages? You can get it from https://download.jitsi.org/stable/
1/may       jitsi-meet_2.0.4546-1_all.deb
10/april    jitsi-meet_2.0.4416-1_all.deb

**Comment by rbdevr** [comment]
I have updated my server to version 17/April (version 2.0.4468-1) it is also working ok. 1/may must have some nasty internal error, memory management or inefficient network transaction going on.

**Comment by damencho** [comment]
There was some jvb logging turned on by mistake, and unstable is updated with a fix. Anyone to test and see whether it is still reproducible there? Just to know whether it is fixed or we need to further work on tracking it ... Thanks.

**Comment by pdarcos** [comment]
Well I just updated my debian server to latest unstable a few minutes ago.
jitsi-videobridge2 2.1-201-g357ee2c3-1
jitsi-meet 2.0.4578-1
jicofo 1.0-579-1
jitsi-meet-web 1.0.4089-1                                                                                                                                                          
jitsi-meet-web-config 1.0.4089-1                                                                                                                                                    
jitsi-meet-prosody 1.0.4089-1      

I noticed right away that it was taking forever to start a meeting (like more than 10 seconds)!

It seems the culprit is lib-jitsi-meet.min.js?v=4089  which takes 4.36 seconds to fetch and the worst is app.bundle.min.js?v=4089 which takes 12.89 seconds to load.

I'm using AWS on c5largex2

**Comment by damencho** [comment]
Downloading is controlled by your nginx config which should not change on upgrade, so I don't see that as a problem in the code.
You can check whether you have https://github.com/jitsi/jitsi-meet/blob/master/doc/debian/jitsi-meet/jitsi-meet.example#L42 in your Nginx config, may speed up downloading.

**Comment by damencho** [comment]
So you see April versions to download faster than those from May, is this the 'Performance/Stability Issue'? Asking to know where to search for the problem, thanks.

**Comment by pdarcos** [comment]
Thanks @damencho That helps a bit but it still takes over 6 seconds to download.
Unfortunately I don't have another vps at another cloud provider to test with and eliminate a possible network issue. It could potentially be a problem with AWS. 
Can someone else test and report back?

**Comment by saghul** [comment]
@pdarcos How large is the download? (wondering if gzip is doing its thing...)

**Comment by pdarcos** [comment]
> So you see April versions to download faster than those from May, is this the 'Performance/Stability Issue'? Asking to know where to search for the problem, thanks.

That was my experience yes. April versions on unstable seemed faster. Not sure it the OP had the same experience since he's running stable and I'm always on unstable (on this AWS instance)

**Comment by pdarcos** [comment]
> @pdarcos How large is the download? (wondering if gzip is doing its thing...)

942 kB

**Comment by pdarcos** [comment]
Shouldn't take 4 or 5 seconds, but this could be AWS being buggy.

Ideally someone would test latest unstable on another provider (or at least not London region) and confirm or deny the "bug"

**Comment by pdarcos** [comment]
Ok, so today it seems it only takes on average 2 to 3 seconds to download the app.bundle so I assume the problems yesterday were due to AWS.

I did notice some new error messages in the console however.

Specifically:

- DevTools failed to load SourceMap: Could not load content for chrome-extension://odbfpeeihdkbihmopkbjmoonfanlbfcl/sourcemaps/contentscript.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME 

- index.js:8 wasm streaming compile failed: TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.
I'm pasting the logs here.

Logs attached.

Thanks
[debian_unstable.log](https://github.com/jitsi/jitsi-meet/files/4598973/debian_unstable.log)
